lib/Runtime/Library/ForInObjectEnumerator.cpp

```diff
@@ -108,30 +100,18 @@ namespace Js
         return firstPrototype;
     }
 
-    BOOL ForInObjectEnumerator::GetCurrentEnumerator()
+    BOOL ForInObjectEnumerator::InitializeCurrentEnumerator()
     {
         Assert(object);
         ScriptContext* scriptContext = GetScriptContext();
-
+        EnumeratorFlags flags = EnumeratorFlags::EnumNonEnumerable | EnumeratorFlags::SnapShotSemantics | (enumSymbols ? EnumeratorFlags::EnumSymbols : EnumeratorFlags::None);
         if (VirtualTableInfo<DynamicObject>::HasVirtualTable(object))
```

Comment:
For now, yes, to avoid the virtual call.  I am going to remove it once I do a JIT fast path for ForInObjectEnumerator